go/vt/vtadmin/cluster/cluster.go

```diff
@@ -633,7 +633,7 @@ func (c *Cluster) getTabletsToQueryForSchemas(ctx context.Context, keyspace stri
 		tabletsToQuery := make([]*vtadminpb.Tablet, 0, len(shards))
 
 		for _, shard := range shards {
-			if !shard.Shard.IsMasterServing {
+			if !(shard.Shard.IsMasterServing && shard.Shard.MasterAlias != nil) {
```

Comment:
Could you add comment for why we need to do this check? Our future selves will be thankful for that :D